docs/source/reference/links.rst

```diff
@@ -47,6 +47,7 @@ For more details, please read :func:`SegNetBasic.predict`.
 .. toctree::
 
    links/segnet
+   experimental/pspnet
```

Comment:
I think we should remove this line because it breaks the correspondence between the tree of modules and the tree of docs.